protocol/communities_messenger_helpers_test.go

```diff
@@ -221,29 +207,56 @@ func newCommunitiesTestMessenger(shh types.Waku, privateKey *ecdsa.PrivateKey, l
 		LinkPreviewRequestEnabled: true,
 		SendStatusUpdates:         true,
 		WalletRootAddress:         types.HexToAddress("0x1122334455667788990011223344556677889900")}
+}
+
+func newTestCommunitiesMessenger(s *suite.Suite, waku types.Waku, config testCommunitiesMessengerConfig) *Messenger {
```

Comment:
should we maybe add `extraOptions []Option` here, like in `newTestMessenger`?
I will definitely use it (have code not commited yet), but I could also add it myself ofc.